contracts/clients/src/signer/getRandomBlsPrivateKey.ts

```diff
@@ -0,0 +1,5 @@
+import { mcl } from "@thehubbleproject/bls";
+
+export default (): string => {
+  return `0x${mcl.randFr().serializeToHexStr()}`;
```

Comment:
I originally put this here because we know that `mcl` has been initialised inside the signer directory.  However, logically I'm thinking now that I should pull this out of the signer.  Curious to hear your thoughts.  Right now if you want to call this you have to create a wallet wrapper first like so.  You probably shouldn't have to create a wallet to get a new PK.

```
const wallet = await this.BlsWalletWrapper(pk);
const newPk = wallet.getRandomBlsPrivateKey();
```

Reply:
Yeah the more that I think about it, I'm gonna move it.